uber/config.py

```diff
@@ -362,6 +362,7 @@ def FINAL_EMAIL_DEADLINE(self):
 
     @property
     def CSRF_TOKEN(self):
+        uber.utils.ensure_csrf_token_exists()
```

Comment:
I don't know why I have to do ```uber.utils``` here, it doesn't like it otherwise.  import order?  it works fine so don't need to change it, just, odd.

Reply:
IMPORTANT NOTE: the original version of this c.CSRF_TOKEN would NOT create a new token if it was ```None```

This new version WILL create the token if one doesn't exist yet.  I think this is the correct behavior